api/models/articles-models.js

```diff
@@ -3,9 +3,13 @@ const format = require("pg-format");
 
 exports.selectArticlesById = (article_id) => {
     let queryString = `
-    SELECT * FROM articles 
-    WHERE article_id = $1;
+    SELECT  COUNT(c.comment_id):: INT AS comment_count, a.* FROM comments c
+    FULL OUTER JOIN articles a 
```

Comment:
Good query overall! I do have a small tweak - I'd recommend using a LEFT JOIN here instead.

The reason for this is that a full outer join includes comments that don't have any articles, so this has a chance of returning comments with no articles. That's a problem for this endpoint, because it's supposed to return just articles.

That's only an issue if there's comments with no articles, which there shouldn't be, but it's better to be safe than sorry.